src/server/hset_family.cc

```diff
@@ -1012,8 +1012,61 @@ int32_t HSetFamily::FieldExpireTime(const DbContext& db_context, const PrimeValu
   }
 }
 
+static std::vector<long> ExpireElements(StringMap* owner, facade::CmdArgList values,
```

Comment:
mutable argument should be the last one - @BorysTheDev .

@vyavdoshenko  do you understand what this function does?